page-front.php

```diff
@@ -33,7 +33,7 @@
 
 			<?php while ( have_posts() ) : the_post(); ?>
 
-			<div id="content" class="content <?php if (is_active_sidebar( 'sidebar' )) { echo "has-sidebar";} ?>">
+			<div id="content" class="content <?php if (is_active_sidebar( 'sidebar' )) { echo 'has-sidebar';} ?>">
```

Comment:
That fixes one problem, but doesn't resolve the other:

https://travis-ci.org/MITLibraries/MITLibraries-child/jobs/139018570#L1369
```
FILE: ...ordpress/src/wp-content/themes/MITLibraries-child/page-front.php
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 1 LINE
----------------------------------------------------------------------
 36 | ERROR | [x] No space after opening parenthesis is prohibited
    |       |     (WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterOpenParenthesis)
 36 | ERROR | [x] No space before closing parenthesis is prohibited
    |       |     (WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBeforeCloseParenthesis)
```

Specifically, the parentheses need spaces on either side - so instead of

```
if (is_active_sidebar( 'sidebar' ))
```

it is looking for

```
if ( is_active_sidebar( 'sidebar' ) )
```